src/doc/examples/mapboxstyle.js

```diff
@@ -1,55 +1,36 @@
 import View from 'ol/View';
-import VectorLayer from 'ol/layer/Vector';
-import VectorSource from 'ol/source/Vector';
-import Feature from 'ol/Feature';
-import { Polygon } from 'ol/geom';
-import { Map, Layer, MapboxLayer, MapboxStyleLayer } from '../../ol';
+import { Map, MapboxLayer, MapboxStyleLayer } from '../../ol';
 import 'ol/ol.css';
 
 export default () => {
   const map = new Map({
     target: 'map',
     view: new View({
-      center: [0, 0],
-      zoom: 1,
+      center: [950690.34, 6003962.67],
+      zoom: 20,
     }),
   });
 
-  const url = `https://maps.geops.io/styles/travic/style.json?key=${window.apiKey}`;
+  const url = `https://maps.geops.io/styles/base_bright_v1/style.json?key=${window.apiKey}`;
+  const mapboxLayer = new MapboxLayer({ url });
 
-  const withoutLabels = new MapboxStyleLayer({
-    mapboxLayer: new MapboxLayer({ url }),
-    visible: false,
-    styleLayersFilter: ({ layout }) => !!layout['text-field'],
+  const poiLayer = new MapboxStyleLayer({
+    name: 'poi layer',
+    visible: true,
+    mapboxLayer,
+    styleLayer: {
+      id: 'poi_with_icons',
+    },
   });
 
-  const labels = new MapboxStyleLayer({
-    mapboxLayer: new MapboxLayer({ url }),
-    visible: false,
-    styleLayersFilter: ({ layout }) => !layout['text-field'],
-  });
+  map.addLayer(mapboxLayer);
+  map.addLayer(poiLayer);
 
-  const polygon = new Layer({
-    olLayer: new VectorLayer({
-      source: new VectorSource({
-        features: [
-          new Feature(
-            new Polygon([
-              [
-                [-1e7, -1e7],
-                [-1e7, 1e7],
-                [1e7, 1e7],
-                [1e7, -1e7],
-                [-1e7, -1e7],
-              ],
-            ]),
-          ),
-        ],
-      }),
-    }),
+  document.getElementById('toggle').addEventListener('click', (evt) => {
+    poiLayer.setVisible(!poiLayer.visible);
+    const { target } = evt;
+    target.innerHTML = `Set service icons ${
```

Comment:
I have used "Set service icons visible/invisible" instead of "Toggle Service Icons" as I think this is clearer. Alternatively, we could use a checkbox.